pkgs/development/python-modules/jmp/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, numpy
+, jax
+, jaxlib
+}:
+
+buildPythonPackage rec {
+  pname = "jmp";
+  version = "0.0.2";
+
+  src = fetchFromGitHub {
+    owner = "deepmind";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-t78zg6H17l/yBPxYygFjPRdclr88BoTDhPUhsEcNUM0=";
+  };
+
+  propagatedBuildInputs = [
+    jax
+    jaxlib
+  ];
+
+  # Test uses unsupported pytest
```

Comment:
What does this mean? Too old?

Reply:
Yeah I believe so, Nix complained about it being deprecated if I remember correctly. I'll take another look.

Reply:
If there is a version constraint then patching it could help.